lib/pages/portal/view/website_view.dart

```diff
@@ -323,33 +397,44 @@ class _WebsiteViewState extends State<WebsiteView> {
 }
 
 class WebsiteIcon extends StatelessWidget {
-  const WebsiteIcon({this.website, this.canEdit, this.size, this.onTap});
+  const WebsiteIcon(
+      {this.website, this.canEdit, this.size, this.image, this.onTap});
 
   final Website website;
   final bool canEdit;
   final double size;
+  final ImageProvider image;
```

Comment:
Add a comment here describing what this does, as it's not super obvious. Something like "If an image is not provided, the corresponding website icon is fetched from the storage, if available."